app/assets/main/react_components/footprint_result/SelectButton.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState }  from 'react';
+import SelectorMultipleTimes from './SelectorMultipleTimes.jsx';
+
+/**
+ * React container for select puttons 
+ */
+const SelectButton = ({selectedMembership, setSelectedMembership, buttonType, text}) => {
+
+    var text = text.split("</span>")
+    var boldText = text[0].replace("<span>","")
+    var unBoldText = text[1]
```

Comment:
Du har säkert kollat på detta, men, finns det något smidigare sätt att göra detta? Har du möjligtvis frågat micke eller någon annan efter tips? Tänker om det finns något smart sätt att extrahera html-taggar från en sträng.

Reply:
Hmm problemet är att jag inte kan ta ut hela strängen i helhet ändå. Måste modda den med att inserta en class, i detta fall className="font-bold". Detta finns inte med i yml-filen. Så även om jag kan inserta en sträng direkt som en tag måste ändå class:en in. Gjorde avvägningen att det skulle ta minst lika många rader kod